Author: gregor herrmann <gregoa@debian.org>
Origin: vendor
Forwarded: no
Last-Update: 2010-06-19
Description: Build.PL looks for Data::UUID >= 1.148, Data::UUID presents its $VERSION as 1.0602

--- a/Build.PL
+++ b/Build.PL
@@ -46,7 +46,7 @@
         object_uuid     => {
             description => "Unique object idenifier using UUIDs",
             requires    => {
-                "Data::UUID" => 1.148
+                "Data::UUID" => '1.010408'
             }
         }
     },
